diff --git a/configure.ac b/configure.ac
index 27751d8..83b400e 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1128,8 +1128,11 @@ AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE,"$GETTEXT_PACKAGE", [The domain to use with g
 AC_SUBST([abs_top_builddir])
 AC_SUBST([builddir])
 
-GNOME_DOC_INIT([0.9.0],,
-  [AC_MSG_WARN([[gnome-doc-utils not found: tutorial will not be built.]])])
+m4_ifdef([GNOME_DOC_INIT], [
+  GNOME_DOC_INIT([0.9.0],,
+    [AC_MSG_WARN([[gnome-doc-utils not found: tutorial will not be built.]])])
+])
+
 
 # ----------------------------------------------------------
 # OUTPUT
